js/releaseDetails.js

```diff
@@ -100,7 +100,7 @@ $(function() {
             if(skipGeneral && i === 0) continue;
             titleTabs.append($("<li></li>").append($("<a></a>")
                 .attr("href", "#" + ids[i])
-                .text(titles[i])));
+                .text(titles[i].substr(0, 40) + (titles[i].length > 40 ? "..." : ""))));
```

Comment:
But I am not moving the header to the tab content, show the `...` will never be shown.